app/templates/drupal/drupal/8.x/composer.json

```diff
@@ -1,17 +1,42 @@
 {
     "name": "client/project",
-    "description": "{Project} drupal codebase for {client}.",
+    "description": "{Project} Drupal codebase for {client}.",
+    "repositories": [
+        {
+            "type": "composer",
+            "url": "https://packagist.drupal-composer.org"
+        }
+    ],
+    "require": {
+        "composer/installers": "^1.0.20",
+        "drupal-composer/drupal-scaffold": "^2.0.1",
+        "cweagans/composer-patches": "~1.0",
+        "drupal/core": "~8.1",
+        "roave/security-advisories": "dev-master"
+    },
     "require-dev": {
         "behat/mink-zombie-driver": "~1.2",
         "drupal/drupal-extension": "~3.0",
         "drush/drush": "^8",
+        "drupal/console": "~0.10",
         "phpmd/phpmd": "~2.1",
-        "drupal/coder": "^8.2",
-        "guzzlehttp/guzzle" : "^6.0@dev",
-        "symfony/dependency-injection": "2.7.*",
-        "symfony/event-dispatcher": "2.7.*"
+        "drupal/coder": "^8.2"
     },
-    "require": {
-        "roave/security-advisories": "dev-master"
+    "conflict": {
+        "drupal/drupal": "*"
+    },
+    "minimum-stability": "dev",
+    "prefer-stable": true,
+    "scripts": {
+        "drupal-scaffold": "DrupalComposer\\DrupalScaffold\\Plugin::scaffold"
+    },
+    "extra": {
+        "installer-paths": {
+            "build/html/core": ["type:drupal-core"],
+            "build/html/modules/contrib/{$name}": ["type:drupal-module"],
+            "build/html/profiles/contrib/{$name}": ["type:drupal-profile"],
+            "build/html/themes/contrib/{$name}": ["type:drupal-theme"],
+            "drush/contrib/{$name}": ["type:drupal-drush"]
+        }
```

Comment:
Shouldn't this be build/html/drush? I seem to recall drush command loading will scan there.